tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -649,11 +671,9 @@ def image_feature_enable(self, **kw):
         Args:
             **kw:
                 imagespec: pool/image_name on which image features will enable
-                image_features: comma seperated value for image features
+                image_feature: comma seperated value for image features, default: journaling
         """
-        cmd = "rbd feature enable {} {}".format(
-            kw.get("imagespec"), kw.get("image_feature")
-        )
+        cmd = f"rbd feature enable {kw.get('imagespec')} {kw.get('image_feature', 'journaling')}"
```

Comment:
This could mean that if we are performing snapshot based mirroring and we don't pass any value to this method, then we might end up enabling journaling feature by default.
Also since this method is specifically for image_feature_enable its logically expected to provide image_feature as input. We should not be taking default value for this mandatory input. 

I would even suggest changing the method definition to
def image_feature_enable(self, image_feature, image_spec, **kw):

Reply:
> Also since this method is specifically for image_feature_enable its logically expected to provide image_feature as input. We should not be taking default value for this mandatory input.
> 
Since this module was specifically formirorring, I don''t see any other image feature in focal point, at least from functional testing pov, so added journaling as default feature for mirror utils.
Removed it considering our plan to optimise rbd and rbd mirror utils

> I would even suggest changing the method definition to def image_feature_enable(self, image_feature, image_spec, **kw):
Thre are 2 modules which are using this method, I would like to keep it as it is from this PR perspective.